test/1_stdlib/StringAPI.swift

```diff
@@ -24,34 +24,44 @@ struct ComparisonTest {
   init(
     _ expectedUnicodeCollation: ExpectedComparisonResult,
     _ lhs: String, _ rhs: String,
-    xfail: TestRunPredicate = .custom({false}, reason: ""),
+    xfail: TestRunPredicate = .never,
     file: String = #file, line: UInt = #line
   ) {
+    self = ComparisonTest(expectedUnicodeCollation, lhs: lhs, rhs: rhs,
+      xfail: xfail, loc: SourceLoc(file, line, comment: "test data"))
+  }
+
+  init(
+    _ expectedUnicodeCollation: ExpectedComparisonResult,
+    lhs: String, rhs: String,
+    xfail: TestRunPredicate, loc: SourceLoc
+  ) {
```

Comment:
Now that this is just a change to a default argument, this initializer isn't needed.